utils/wasm-gen/src/generator/syscalls.rs

```diff
@@ -239,68 +239,9 @@ impl InvocableSysCall {
     // If syscall changes from fallible into infallible or vice versa in future,
     // we'll see it by analyzing code coverage stats produced by fuzzer.
     pub(crate) fn is_fallible(&self) -> bool {
-        let underlying_syscall = match *self {
-            Self::Loose(sc) => sc,
-            Self::Precise(sc) => sc,
-        };
-
-        match underlying_syscall {
-            SysCallName::EnvVars
-            | SysCallName::BlockHeight
-            | SysCallName::BlockTimestamp
-            | SysCallName::Debug
-            | SysCallName::Panic
-            | SysCallName::OomPanic
-            | SysCallName::Exit
-            | SysCallName::GasAvailable
-            | SysCallName::Leave
-            | SysCallName::MessageId
-            | SysCallName::ProgramId
-            | SysCallName::Random
-            | SysCallName::Size
-            | SysCallName::Source
-            | SysCallName::ValueAvailable
-            | SysCallName::Value
-            | SysCallName::WaitFor
-            | SysCallName::WaitUpTo
-            | SysCallName::Wait
-            | SysCallName::Alloc
-            | SysCallName::Free
```

Comment:
`SysCallName::Alloc` and `SysCallName::Free` must be fallible too

Reply:
But they don't write error to the pointer, so it depends on the definition of `fallible` in current context